src/components/SwipeButtons.tsx

```diff
@@ -46,4 +44,6 @@ const styles = StyleSheet.create({
         shadowRadius: 6.46,
         elevation: 9,
     },
-})
\ No newline at end of file
+})
+
+export default SwipeButtons;
```

Comment:
If we change SwipeButtons to be export default above then we can remove this line.